app/helpers/messages_helper.rb

```diff
@@ -57,7 +57,9 @@ def message_presentation(message)
     when "sound"
       message_sound_presentation(message)
     else
-      auto_link h(ContentFilters::TextMessagePresentationFilters.apply(message.body.body)), html: { target: "_blank" }
+      # Render the full ActionText content to preserve unfurled links and other attachments
+      # Use ActionText's built-in rendering
+      message.body.to_s.html_safe
```

Comment:
I'm somewhat concerned about calling .html_safe on user inputs. The TextMessagePresentationFilters did a few things like sanitize the input, and style the unfurled links.